### PR TITLE
Final changes

### PR DESCRIPTION
**Pull request by mthembbu**

### CODE REVIEW

File: skietbaan_fe/src/actions/competition.action.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
-import { NEW_COMP, FETCH_COMP, UPDATE_COMP_STATUS, URL, LOCAL_URL } from './types';
+import { NEW_COMP,
```
Comment (FredJEhlers) [inline skietbaan_fe/src/actions/competition.action.js]: Change it so each line has its own import, please.
```
import {
	NEW_COMP,
	FETCH_COMP,
	UPDATE_COMP_STATUS,
	URL 
} from './types';
```

File: skietbaan_fe/src/components/CreateComp.js [hunk 2]
```diff
@@ -1,8 +1,5 @@
 import React, { Component } from 'react';
-import { Form,
-		Button,
-		Row,
-		Col} from 'react-bootstrap';
+import { Form, Button, Row, Col } from 'react-bootstrap';
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/CreateComp.js]: Each on its own line please like above

File: skietbaan_fe/src/scss/createcomp.css [hunk 3]
```diff
@@ -3,14 +3,40 @@
...
+  border-radius: 25px;
+  font-size: 16px;
+  font: helvetica;
+  font-weight: lighter;
+  margin-top: 20px;
+  color: white;
+}
+
+.comp-submit-btn-constainer{
+      padding: 20px;
```
Comment (FredJEhlers) [inline skietbaan_fe/src/scss/createcomp.css]: inconsistent padding

Comment (mthembbu) [reply skietbaan_fe/src/scss/createcomp.css]: It's due a design from the wireframes, the button needs to be further away from the inputs

**Pull request merged**